src/base/noscript.scss

```diff
@@ -8,6 +8,12 @@
 	display: block !important;
 }
 
+@media screen {
+	%noscript-screen-display-block-important {
```

Comment:
Can you create a simplified version of this on Codepen or similar. I don't think it's right having a media query with no actual CSS, plus sharing placeholder accros media queries shouldn't work